### PR TITLE
Fix the CI matrix to actually run the testSuite with PhantomJS. -_-'

### PR DESCRIPTION
**Pull request by sjrd**

### CODE REVIEW

File: js-envs/src/main/scala/org/scalajs/jsenv/RetryingComJSEnv.scala [hunk 1]
```diff
@@ -31,8 +31,8 @@ import scala.util.{Try, Failure, Success}
  *
  *  No retrying is performed for synchronous, or normal asynchronous runs.
  */
-final class RetryingComJSEnv(baseEnv: ComJSEnv,
-    maxRetries: Int) extends ComJSEnv {
+final class RetryingComJSEnv(val baseEnv: ComJSEnv,
+    val maxRetries: Int) extends ComJSEnv {
```
Comment (gzm0) [inline js-envs/src/main/scala/org/scalajs/jsenv/RetryingComJSEnv.scala]: Why this?

Comment (gzm0) [inline js-envs/src/main/scala/org/scalajs/jsenv/RetryingComJSEnv.scala]: Oh, I see...

File: project/Build.scala [hunk 2]
```diff
@@ -853,10 +857,16 @@ object Build extends sbt.Build {
 
           case _: PhantomJSEnv =>
             Seq(Tests.Argument("-tphantomjs"))
+
+          case env: RetryingComJSEnv =>
+            envTagsFor(env.baseEnv)
+
           case _ =>
```
Comment (gzm0) [inline project/Build.scala]: Maybe we should fail in this case, just so we are sure we update it.

Comment (sjrd) [inline project/Build.scala]: Yep, good idea. I'll first see whether the CI is going anywhere with the current commit.

Comment (sjrd) [inline project/Build.scala]: Done.

**Comment by sjrd** [comment]
Updated.

**Comment by gzm0** [comment]
LGTM

**Pull request merged**